hadoop-ozone/dist/src/main/compose/ozonesecure-ha/test.sh

```diff
@@ -34,6 +34,9 @@ execute_robot_test ${SCM} freon
 execute_robot_test ${SCM} basic/links.robot
 
 execute_robot_test ${SCM} s3
+
+export SCM=scm2.org
+execute_robot_test ${SCM} security-ha
```

Comment:
Instead of introducing a new test suite (`security-ha`) with only a single test case that is not even HA-specific, I think we should run the complete `admincli` test suite (probably for both scm1 and scm2).

```suggestion
execute_robot_test ${SCM} admincli

export SCM=scm2.org
execute_robot_test ${SCM} admincli
```

Only one test case in each `admincli` test file needs to be changed for this to work: `... on unknown host` fails because `--scm` parameter is ignored for SCM HA cluster.  I think it is a bug that should be fixed in the long term.  But for now these test cases can be disabled (commented out), as they are not too important.  I think it is more important to have full `admincli` test coverage for HA than to run this marginal test case for non-HA.

https://github.com/apache/ozone/blob/49538aaa11f41bf80f565f9ce4f2c49ebac5fd51/hadoop-ozone/dist/src/main/smoketest/admincli/container.robot#L75-L77

Reply:
Done. For now, commented them.

Reply:
For ha --scm should it pass multiple scms? What is the expected behavior, I can open a Jira for this.

Reply:
I guess SCM service ID and SCM host should be exclusive, only one of these should be allowed.

https://github.com/apache/ozone/blob/03620da6158534ac618354737a8b35d09a25fa33/hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ScmOption.java#L46-L53